res/css/structures/_RoomDirectory.scss

```diff
@@ -45,22 +64,17 @@ limitations under the License.
 }
 
 .mx_RoomDirectory_listheader {
-    display: table;
-    table-layout: fixed;
-    width: 100%;
+    display: flex;
     margin-top: 12px;
     margin-bottom: 12px;
-    border-spacing: 5px;
 }
 
 .mx_RoomDirectory_searchbox {
-    display: table-cell;
-    vertical-align: middle;
+    flex: 1 !important;
```

Comment:
do more specific styles help at all in this case? Something like `.mx_RoomDirectory_dialogWrapper .mx_RoomDirectory_searchBox` ?

Reply:
Tried that, but the rule it's trying to override is very very specific (see `_dharma.scss:260`). The whole `mx_textinput`/ global `input` styles need to be better scoped anyways, so rather postpone this till then.

Reply:
:(

+1 to postponing. It's not a major problem, just a thing we should avoid if possible